lib/battle/CBattleInfoCallback.cpp

```diff
@@ -855,6 +855,24 @@ AccessibilityInfo CBattleInfoCallback::getAccesibility() const
 		ret[BattleHex(0, y)] = EAccessibility::SIDE_COLUMN;
 	}
 
+	//special battlefields with logically unavailable tiles
+	std::vector<BattleHex> impassableHexes;
+	if(battleGetBattlefieldType().num == BFieldType::SHIP_TO_SHIP)
+		impassableHexes = 
+		{ 

```

Comment:
@dydzio0614
I think this is totally fine to avoid brackets if body is a single-line expression and especially so if it's not really followed by any code because this is look worse:
```
if(a)
	for(auto item : group)
		doSth();
else if
{
	code();
}
else
{
	code();
}
```
I sadly can't find any good example of single, but multi-line expressions in our code. Most of them are just bad code so I would rather not put them as example for anyone.